dashboard/src/shared/Config.test.ts

```diff
@@ -31,7 +31,6 @@ describe("Config", () => {
   });
 
   it("does not returns the overriden namespace if NODE_ENV=production", async () => {
-    process.env.NODE_ENV = "production";
```

Comment:
Guessing there was some other reason for removing this, but it appears to make this contradict the previous one above, yet they both pass?